doc/source/dev/conduct/report_handling_manual.rst

```diff
@@ -0,0 +1,171 @@
+.. _CoC_reporting_manual:
+
+SciPy Code of Conduct - How to follow up on a report
+----------------------------------------------------
+
+This is the manual followed by SciPy's Code of Conduct Committee. It's used
+when we respond to an issue to make sure we're consistent and fair.
+
+Enforcing the Code of Conduct impacts our community today and for the future.
+It's an action that we do not take lightly. When reviewing enforcement
+measures, the Code of Conduct Committee will keep the following values and
+guidelines in mind:
+
+* Act in a personal manner rather than impersonal.  The Committee can engage
+  the parties to understand the situation, while respecting the privacy and any
+  necessary confidentiality of reporters.  However, sometimes it is necessary
+  to communicate with one or more individuals directly: the Committee's goal is
+  to improve the health of our community rather than only produce a formal
+  decision.
+
+* Emphasize empathy for individuals rather than judging behavior, avoiding
+  binary labels of "good" and "bad/evil". Overt, clear-cut aggression and
+  harassment exists and will be addressed unambiguously.  But many scenarios
+  that can prove challenging to resolve are those where normal disagreements
+  devolve into inappropriate behavior from multiple parties.  Understanding the
```

Comment:
> "inappropriate" is an ugly word, because it begs the question as to who decides what is "appropriate", and what basis do they use for deciding? 

The same people who determine what is unhelpful/harmful? 

I'm taking over your terminology, but please keep in mind that the qualifications you use (ugly, pompous, threatening) are quite opinionated and won't help your case if you'd use them with people who know you less well.